generate_doc.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+export SECRETS_FOR_DYNACONF='mobilizon_reshare/.secrets.toml'
+poetry run sphinx-apidoc -f -o api_documentation/source/ mobilizon_reshare/
+cd api_documentation 
+poetry run make html
+if [ -z $1 ] || [[ $1 != 'nox' ]]
+then
+#	xdg-open build/html/index.html
+	firefox build/html/index.html
+fi
+cd ..
```

Comment:
Can we avoid this? Or just move it to another script as we need this to work also in CI